dll/win32/shell32/droptargets/CFSDropTarget.cpp

```diff
@@ -83,12 +83,26 @@ HRESULT CFSDropTarget::_CopyItems(IShellFolder * pSFFrom, UINT cidl,
     if (!pszSrcList)
         return E_OUTOFMEMORY;
 
+    BOOL bRenameOnCollision = FALSE;
+    if (bCopy)
+    {
+        WCHAR szPath1[MAX_PATH], szPath2[MAX_PATH];
+        lstrcpynW(szPath1, pszSrcList, _countof(szPath1));
+        PathRemoveFileSpecW(szPath1);
+        lstrcpynW(szPath2, m_sPathTarget, _countof(szPath2));
+        PathRemoveFileSpecW(szPath2);
+        if (lstrcmpiW(szPath1, szPath2) == 0)
```

Comment:
Maybe the following may be more efficient and less stack-space-consuming:
```suggestion
        PCWSTR pszFileName1 = PathFindFileNameW(pszSrcList);
        PCWSTR pszFileName2 = PathFindFileNameW(m_sPathTarget);
        if (lstrcmpiW(pszFileName1, pszFileName2) == 0)
```

Reply:
Did you think the case that two directories are different and two file titles are the same?